tests-ui/tests/slow/groupNode.test.ts

```diff
@@ -362,7 +362,7 @@ describe('group node', () => {
     const { ez, graph, app } = await start()
```

Comment:
We can probably remove the legacy slow tests later as they are currently disabled anyway. No need to further maintain them.